### PR TITLE
libvirt: add function to create tpm device

### PR DESCRIPTION
**Pull request by Yingshun**
This function is to create tpm dev.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -2525,6 +2526,34 @@ def update_memballoon_xml(vmxml, membal_dict):
     vmxml.sync()
 
 
+def create_tpm_xml(params):
+    """
+    Create xml for tpm device
+
+    :param params: tpm parameter dict
+    :return: tpm device
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: return tpm device xml

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -2525,6 +2526,34 @@ def update_memballoon_xml(vmxml, membal_dict):
...
+        tpm_backend = tpm_dev.Backend()
+        tpm_backend.backend_type = backend_type
+        if backend_version:
+            tpm_backend.backend_version = backend_version
+        if encryption_secret:
+            tpm_backend.encryption_secret = encryption_secret
+        if device_path:
+            tpm_backend.device_path = device_path
+        tpm_dev.backend = tpm_backend
+    return tpm_dev
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: the return value is tpm_dev.xml or tpm_dev?

Comment (Yingshun) [reply virttest/utils_test/libvirt.py]: tpm_dev

Comment (chunfuwen) [reply virttest/utils_test/libvirt.py]: the method name is :create_tpm_xml ,suggest syn up with return value to avoid confusion

Comment (Yingshun) [reply virttest/utils_test/libvirt.py]: updated。

**Pull request merged**